experimental/kubernetes/README.md

```diff
@@ -0,0 +1,186 @@
+# Spinnaker on Kubernetes
+
+This is an experimental example of how to run Spinnaker on a Kubernetes cluster using docker images
+of Spinnaker components.
+
+It requires you to have the [Google Cloud SDK](https://cloud.google.com/sdk/#Quick_Start) installed with the `kubectl` component
+
+  ```
+  $ gcloud components install kubectl
+  ```
+
+## Limitations
+
+Both storage mechanisms (Cassandra and Redis) are mostly out-of-the-box, which is likely nowhere
+near production grade.
+
+## TODOs
+
+1. Cassadra deployment options:
+    a. Create own Dockerfile that executes the keyspace creation scripts (how?).
+    b. Use embedded/in-memory Cassandra (like docker-compose deployment)
+    c. Remove Cassandra dependency altogether
+1. Add instructions for making a persistent disk (PD) for the configuration. This makes >1 node much
+ easier to configure.
+
+
+# Instructions
+
+1. (Optional) Set the default zone and project. All `gcloud` commands below will require the `--zone`
+parameter if you skip this step.
+
+  ```
+  $ gcloud config set compute/zone us-central1-f
+  ```
+
+
+## Cluster Creation
+
+1. Create a new Kubernetes cluster running on Google Container Engine. This tutorial does a lot of
+copying files to the node, so for simplicity, we will only use 1 host.
+
+  ```
+  $ gcloud container clusters create my-spinnaker-on-kubernetes --num-nodes 1 --scopes storage-rw,compute-rw
+  ```
+
+1. Get the name of the node in a variable for convenience.
+  ```
+  $ MY_GKE_NODE=`kubectl get nodes -o go-template='{{ (index .items 0).metadata.name }}'`
+  ```
+
+
+## Spinnaker Configuration
+
+1. Create a directory on your cluster node to store the configuration.
+
+  ```
+  $ gcloud compute ssh root@$MY_GKE_NODE 'mkdir -p /root/.spinnaker'
+  ```
+
+1. Copy the contents of the `../../config` directory to the node.
```

Comment:
need to make the prerequisites on configuring this directory clearer. I had to spell it out for the docker compose configuration as this was confusing people.